src/doc-components/GlobalVariable/globalVariable.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+import data from "../../data/globalVariables.yml"
```

Comment:
Can't we use the data provider instead? Look at my data provider PR for example.

Reply:
yes, but I didn't want to introduce merge conflicts since you were renaming files. We can create a follow up ticket to re-factor.